npm/css-src/sass/_success_summary.scss

```diff
@@ -0,0 +1,105 @@
+.success-summary {
```

Comment:
Could you add a comment to this to file linking to the design system code that we adapted? It'll be a helpful reference if we have to make more changes in future.

Reply:
I've added a comment in.